src/index.ts

```diff
@@ -216,16 +206,9 @@ export class ActionsAPI {
    * @param source - Source path of the file
    * @param dest - Destination path of the file.
    */
-  async copyFile(
-    source: string,
-    dest: string
-  ): Promise<any> {
+  async copyFile(source: string, dest: string): Promise<any> {
```

Comment:
On all of these file io functions should we be checking to make sure the path / file is valid?